src/Serilog.Sinks.AzureDataExplorer/AzureDataExplorerSinkOptions.cs

```diff
@@ -78,30 +120,43 @@ public int QueueSizeLimit
         public string ApplicationKey { get; private set; }
         public X509Certificate2 ApplicationCertificate { get; private set; }
         public string Authority { get; private set; }
-        public bool SendX5c { get; private set; }
+        public bool SendX5C { get; private set; }
         public string AzureRegion { get; private set; }
         public Azure.Core.TokenCredential TokenCredential { get; private set; }
 
         public AzureDataExplorerSinkOptions()
         {
-            Period = TimeSpan.FromSeconds(10);
-            BatchPostingLimit = 1000;
-            QueueSizeLimit = 100000;
+            this.Period = TimeSpan.FromSeconds(10);
+            this.BatchPostingLimit = 1000;
```

Comment:
If we are using this default, this needs to be added into the README

Reply:
already there